packages/angular_devkit/build_angular/src/angular-cli-files/utilities/package-chunk-sort.ts

```diff
@@ -29,5 +29,13 @@ export function generateEntryPoints(
 
   entryPoints.push('main');
 
+  const duplicateBundles = entryPoints.find(x => (
```

Comment:
This would only catch `main` since the loop above prevents double inclusion of any other name.

Reply:
updated.